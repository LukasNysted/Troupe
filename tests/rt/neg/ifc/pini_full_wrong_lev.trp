let val a_ = attenuate (authority, {alice})
    val c = pinipushto (a_ , {bob}) 
    val _ = if (1 raisedTo {alice, bob}) then () else ()
    val c2 = pinipushto(a_, {})
    val _ = debugpc ()
    val _ = pinipop (c)
in debugpc ()
end