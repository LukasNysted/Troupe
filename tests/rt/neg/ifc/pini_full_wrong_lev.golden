2020-03-26T16:43:12.945Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
[31mRuntime error in thread 138d4c0b-1784-4d9a-8bf4-eb4255143b3c@{}%{}[39m
[31m>> level argument of pinipushto operation must flow to the current blocking level[39m
[31m| the blocking level: {alice,bob}[39m
[31m| the level argument: {}@{}%{}[39m
